docs/src/userguide/wrapping_CPlusPlus.rst

```diff
@@ -81,8 +81,8 @@ and the implementation in the file called :file:`Rectangle.cpp`:
         }
 
         void Rectangle::getSize(int *width, int *height) {
-            width = x1 - x0;
-            height = y1 - y0;
+            (*width) = x1 - x0;
```

Comment:
The parenthesis are not needed, but the * is needed in order to stored the result in the integer pointed by width and height.  
As it was previously, an error was raised.